src/actions/LocalNotification.ts

```diff
@@ -0,0 +1,22 @@
+import PushNotification from 'react-native-push-notification';
+
+const LocalNotification = () => {
```

Comment:
- Can we export this directly instead of using the default export? It would be easy to find references to LocalNotification once they are used in multiple places.